src/main/scala/tile/BusErrorUnit.scala

```diff
@@ -115,19 +113,19 @@ class BusErrorUnit[T <: BusErrors](t: => T, params: BusErrorUnitParams)(implicit
     def numberRegs(x: Seq[Seq[RegField]]) = x.zipWithIndex.map {case (f, i) => (i * regWidth / 8) -> f }
 
     val omRegMap = node.regmap(numberRegs(Seq(
-      reg(cause, "cause", cause_desc),
-      reg(value, "value", value_desc),
+      reg(cause.asUInt, "cause", cause_desc),
+      reg(value.asUInt, "value", value_desc),
```

Comment:
```suggestion
      reg(cause, "cause", cause_desc),
      reg(value, "value", value_desc),
```

Reply:
If we remove ".asUInt", there will be a type mismatch